EventSubscriber/EventSubscriberInterface.php

```diff
@@ -2,8 +2,12 @@
 
 namespace Doctrine\Bundle\DoctrineBundle\EventSubscriber;
 
+use Doctrine\Bundle\DoctrineBundle\Attribute\AsDoctrineListener;
 use Doctrine\Common\EventSubscriber;
 
+/**
+ * @deprecated use the {@link AsDoctrineListener} attribute instead
```

Comment:
I think this should be `@see` rather than `@link` here (`@link` is about linking to a URL)